### ISSUE TITLE
[PLAIN LANGUAGE]: The bot should have an active voice, and not talk about itself in third person

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#25184]
# [508-defect-3](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-3)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** _Trevor_

## Details

The third message from the bot on the first load says "Ask it a question". Who is it? It would be good to review this content and determine if the bot is going to take a first-person voice, or a third-person, and be clear who "it" is.

## Acceptance Criteria

- [ ] Users can easily establish they're talking to a bot OR
- [ ] If the bot is going to take a third-person voice, make sure that voice is consistent

## Environment

* https://staging.va.gov/virtual-agent/

## Screenshots or Trace Logs

![Screen Shot 2021-05-24 at 4 52 31 PM](https://user-images.githubusercontent.com/934879/119412021-eb20e680-bcb0-11eb-9a38-b4d822f30f72.png)

**Comment by ianpsant** [comment]
Custom - Within our control and we are able to change. Working with the content team to review for consistency. We can track this accessibility issue and address if PVA development moves forward towards production.

**Comment by ianpsant** [comment]
We have since reviewed all Content with the the Content team and removed responses in the passive voice. The Content Team is reviewing all new responses moving forward as well.
